src/Proto3/Wire/Decode.hs

```diff
@@ -408,6 +410,22 @@ enum = fmap toEither parseVarInt
 packedVarints :: Integral a => Parser RawPrimitive [a]
 packedVarints = fmap (fmap fromIntegral) (runGetPacked (many getBase128Varint))
 
+getBase128Varint :: Get Word64
+getBase128Varint = loop 0 0
```

Comment:
Does this function do something that `takeVI'` does not?  I'm wondering why it is still here.

Reply:
It's in the Get monad, and I didn't want to figure out how to embed takeVI' into the get monad. But it would be an improvement to do so, certainly.

Reply:
OK.